worldengine/generation.py

```diff
@@ -124,12 +124,17 @@ def initialize_ocean_and_thresholds(world, ocean_level=1.0):
     """
     e = world.elevation['data']
     ocean = fill_ocean(e, ocean_level)
-    hl = find_threshold_f(e, 0.10)
-    ml = find_threshold_f(e, 0.03)
+    pl = find_threshold_f(e, 0.70, ocean=ocean)  # the highest 70% of land are declared plains
+    hl = find_threshold_f(e, 0.35, ocean=ocean)  # the highest 35% are declared hills
+    ml = find_threshold_f(e, 0.10, ocean=ocean)  # the highest 10% are declared low mountains
+    mml = find_threshold_f(e, 0.06, ocean=ocean)  # the highest 10% are declared medium mountains
```

Comment:
You are right. I will do that once the values are optimized a bit. I will have to touch those lines anyway.